.github/workflows/file.yml

```diff
@@ -12,9 +12,9 @@ on:
       - '.github/workflows/analyze.yml'
       - '.github/actions/dart-test/**'
       - '.github/actions/coverage/**'
-      - 'dart/**'
-      - 'flutter/**'
-      - 'file/**'
+      - 'packages/dart/**'
+      - 'packages/flutter/**'
```

Comment:
Think we don't need flutter here, since we only depend on dart?

Reply:
yeah we can remove it